modules/nw-ingress-setting-select-subnet-LoadBalancerService.adoc

```diff
@@ -0,0 +1,157 @@
+//
+// * ingress/configure-ingress-operator.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="nw-ingress-setting-select-subnet-LoadBalancerService_{context}"]
+= Choosing subnets while creating a LoadBalancerService ingress controller
+You can manually specify load balancer subnets for Ingress Controllers in an existing cluster. By default, load balancer subnets are automatically discovered by AWS, but specifying them in the Ingress Controller  overrides this, allowing for manual control.
+
+[NOTE]
+====
+This feature and procedure applies only to AWS. 
+====
+
+.Prerequisites
+* You must have an installed AWS cluster.
+* You must know the names or IDs of the subnets to which you intend to map your IngressController.
+
+.Procedure
+To create an Ingress Controller with manually specified load balancer subnets, you can follow these steps:
+
+. Create a custom resource (CR) file.
+.. Create a YAML file (e.g., `sample-ingress.yaml`) with the following content:
++
+[source,yaml]
+----
+apiVersion: operator.openshift.io/v1
+kind: IngressController
+metadata:
+  namespace: openshift-ingress-operator
+  name: <name>
+spec:
+  domain: <domain>
+  endpointPublishingStrategy:
+    type: LoadBalancerService
+    loadBalancer:
+      scope: External
+  dnsManagementPolicy: Managed
+----
+
+. Create a custom resource (CR) file.
+
+.. Create a YAML file (e.g., `sample-ingress.yaml`) with the following content:
++
+[source,yaml]
+----
+apiVersion: operator.openshift.io/v1
+kind: IngressController
+metadata:
+  name:  <name> <1>
+  namespace: openshift-ingress-operator
+spec:
+  domain: <domain> <2>
+  endpointPublishingStrategy:
+    type: LoadBalancerService
+    loadBalancer:
+      scope: External
+      providerParameters:
+        type: AWS
+        aws:
+          type: Classic
+          classicLoadBalancer: <3>
+            subnets:
+              ids: <4>
+              - <subnet> <5>
+              - <subnet>
+              - <subnet>
+----
+<1> Replace `<name>` with a name for the Ingress Controller
+<2> Replace `<domain>` with the DNS name serviced by the Ingress Controller
+<3> You can also use the `networkLoadBalancer` field if using an NLB
+<4> You can optionally specify subnet by name using the `names` field
+<5> Specify subnet IDs (or names if you using `names`).
++
+[IMPORTANT]
+====
+You can specify a maximum of one subnet per availability zone. Only provide public subnets for external Ingress Controllers and private subnets for internal Ingress Controllers.
+====
++
+.. Apply the CR file:
+
+. Save the file and apply it using the OpenShift CLI:
++
+[source,terminal]
+----
+$  oc apply -f sample-ingress.yaml
+----
+.. Confirm the load balancer was provisioned successfully by checking the IngressController conditions. 
++
+[source,terminal]
+----
+$ oc get ingresscontroller -n openshift-ingress-operator <name> -o jsonpath="{.status.conditions}" | yq -PC
+----
++
+= Updating the subnets on an existing Ingress Controller
```

Comment:
You need a unique ID before this header: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#anchoring-in-module-files

Also, I think this should be a `==`, since you already use the `=` in your first heading. I'm surprised this didn't throw an asciidoc preview error. If you run `asciibinder build`, you'll probably see it come up but you're only supposed to have one H1 in a module file.

 If you want this to be an H1, you could break it out into its own procedure module, which may honestly be a good option for this since its quite a long procedure module anyways, so for modular docs you may just want to go with this option. Because additionally, I'm thinking that this heading should have a little introductory text to it as well, which sorta reinforces my thinking that this should just be its own module.

Reply:
Done! Thank you!